### DISCUSSION TITLE
Fishing spots not showing up in Runelite w/ 117 HD plugin

### DISCUSSION THREAD

**Discussion by dorgaorfa** [runelite/runelite#16787] [Q&A]
Hello everyone. New player here. When I use the 117 HD plugin my fishing spots are not shown on my screen. Below is an example.

![image](https://github.com/runelite/runelite/assets/139558840/66f9b2d9-41e8-485a-9b1c-aa02fc8d1bc8)

I only use two plugins - 117HD and Quest Helper. If you need any more information that may help me please let me know, and thanks in advanced to anyone who can help me solve this issue.

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.

**Reply by dorgaorfa** [reply]
Thank you! I'll do just that.
